R-packages/evalcast/R/error_measures.R

```diff
@@ -99,6 +99,94 @@ interval_coverage <- function(coverage) {
   }
 }
 
+#' Overprediction component of the weighted interval score
+#' 
+#' Requires symmetric quantile forecasts. Roughly, a penalty for predicted 
+#' quantiles smaller than .5 falling above the observed value.
+#'
+#' @param quantile vector of forecasted quantiles
+#' @param value vector of forecasted values
+#' @param actual_value Actual value.
+#' 
+#' @export
+overprediction <- function(quantile, value, actual_value) {
+  score_func_param_checker(quantile, value, actual_value, "overprediction")
+  if (!is_symmetric(quantile)) {
+    warning(paste0("overprediction/underprediction/sharpness require",
+                   "symmetric quantile forecasts. Using NA."))
+    return(NA)
+  }
+  if (all(is.na(actual_value))) return(NA)
+  actual_value <- unique(actual_value)
```

Comment:
Why are we going anything if `actual_value` contains multiple values?

Reply:
We don't. Multiple values get killed by `score_func_param_checker()` on 113.